test/e2e/network/netpol/test_helper.go

```diff
@@ -18,14 +18,15 @@ package netpol
 
 import (
 	"context"
-	"time"
-
+	"fmt"
 	"github.com/onsi/ginkgo"
+	"github.com/pkg/errors"
```

Comment:
there is this:
https://github.com/kubernetes/kubernetes/issues/103043

line 107 could be fmt.Errorf(...) to avoid the import.